nixui/options/nix_eval.py

```diff
@@ -4,11 +4,19 @@
 import importlib.resources
 from contextlib import contextmanager
 from string import Template
+import os
+import sys
 
 from nixui.utils.logger import logger
 from nixui.utils import cache
 from nixui.options.attribute import Attribute
 
+env_nix_instantiate = os.environ.copy()
+env_nix_instantiate["NIXPKGS_ALLOW_UNFREE"] = "1"
```

Comment:
Isn't `NIXPKGS_ALLOW_UNFREE` inherited from the environment in which Nix-Gui is run?

Reply:
yes, thats the problem. when NIXPKGS_ALLOW_UNFREE is false, the nix-instantiate command throws, and nix-gui fails to parse the invalid json